app/src/main/java/com/mapbox/maps/testapp/examples/MapViewCustomizationActivity.kt

```diff
@@ -62,7 +63,7 @@ class MapViewCustomizationActivity : AppCompatActivity() {
 
   private fun configureMapViewFromXml() {
     // let's set `custom` token to MapView from code (however it will be same token from resources so that map will work)
-    MapboxOptions.setDefaultResourceOptions(this, getString(R.string.mapbox_access_token))
+    CredentialsManager.shared = CredentialsManager(getString(R.string.mapbox_access_token))
```

Comment:
By the way it looks more like there was a bug: method is named `configureMapViewFromXml` and we're setting `CredentialsManager.shared` when actually map view from xml is already inflated with some token.
Can we rebuild that example showcasing following scenario:
 - map view from xml uses default token (it goes from resources)
 - map view created programatically in `configureMapViewFromCode` uses some custom token

Reply:
Actually, we're already doing it via `.accessToken(getString(R.string.mapbox_access_token))` when creating `resourceOptions`. Then I guess we should either remove this line or use it __before__ `setContentView` and state clearly in comment that we're overriding default `CredentialsManager.shared` meaning all map views inflated from XML will use token coming not from resources file.